<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>entering_signin_form</class>
 <widget class="QDialog" name="entering_signin_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>520</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>520</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>User agent</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Scheme</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Region</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Use dual stack</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max connections</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Request timeout Ms</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect timeout MS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Endpoint Override</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Proxy host</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Proxy port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Proxy username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Proxy password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Verify SSL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>CA Path</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>User Agent string user for http calls. This is filled in for you in the constructor. Don't override this unless you have a really good reason. </string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Http scheme to use. </string>
   </property>
   <item>
    <property name="text">
     <string>HTTPS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HTTP</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>AWS Region to use in signing requests.</string>
   </property>
   <property name="text">
    <string>US_EAST_1</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>80</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use dual stack endpoint in the endpoint calculation. It is your responsibility to verify that the service supports ipv6 in the region you select. </string>
   </property>
   <item>
    <property name="text">
     <string>-</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TRUE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FALSE</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Max concurrent tcp connections for a single http client to use.</string>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Socket read timeouts. This should be more than adequate for most services. However, if you are transfering large amounts of data or are worried about higher latencies, you should set to something that makes more sense for your use case. </string>
   </property>
   <property name="suffix">
    <string>ms</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>3000</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>140</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Socket connect timeout. Unless you are very far away from your the data center you are talking to. 1000ms is more than sufficient.</string>
   </property>
   <property name="suffix">
    <string>ms</string>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="singleStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>override the http endpoint used to talk to a service. Use this in conjunction with authenticationRegion. </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If you have users going through a proxy, set the host here. </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If you have users going through a proxy, set the port here. </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If you have users going through a proxy, set the username here. </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If you have users going through a proxy, set the password here. </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>260</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If you need to test and want to get around TLS validation errors, do that here. you probably shouldn't use this flag in a production scenario. </string>
   </property>
   <item>
    <property name="text">
     <string>FALSE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TRUE</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>If your Certificate Authority path is different from the default, you can tell curl where to find your CA trust store. </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Payload Signing Policy</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>161</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Virtual Addressing</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>360</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>whether to sign the body of the request or not.</string>
   </property>
   <item>
    <property name="text">
     <string>Never</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Request dependent</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Always</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>380</y>
     <width>86</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>TRUE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FALSE</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>320</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>340</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
